src/test/scala/tech/sourced/gemini/CassandraSparkSpec.scala

```diff
@@ -172,15 +172,34 @@ class CassandraSparkSpec extends FlatSpec
       .start()
 
     val gemini = Gemini(sparkSession, logger, SIMILARITIES)
-
-    val file = new File("src/test/resources/models.py")
     val channel = ManagedChannelBuilder.forAddress("localhost", server.getPort).usePlaintext(true).build()
     val bblfshStub = BblfshClient("localhost", server.getPort)
     val feStub = FeatureExtractorGrpc.stub(channel)
 
-    val similar =
```

Comment:
Do you think it could be worth to refactor this single test to two separate ones now?

i.e something like
 - `"Query for duplicate files" should "return 1 file"`
 - `"Query for similar files" should "return 1 file"`

Asking, as I find useful having each test to test only a single case/behaviour.